protos/bc.proto

```diff
@@ -24,24 +24,32 @@ message RpcTransaction {
 message RpcFeedTransaction {
     string owner_addr = 1;
     string feed_addr = 2;
-    string amount = 3;
-    string tx_fee = 4;
-    string private_key_hex = 5;
+    string data_type = 3;
```

Comment:
Guessing from the name, this should better by [an enumeration](https://developers.google.com/protocol-buffers/docs/reference/javascript-generated#enum)